buildpack.md

```diff
@@ -174,14 +174,15 @@ Executable: `<layers>/<layer>/exec.d/<process>/<executable>`, Working Dir: `<app
 | Input             | Description
 |-------------------|----------------------------------------------
 | `$0`              | Absolute path of the executable
-| FD 3              | A third open file [†](README.md#linux-only)descriptor or [‡](README.md#windows-only)handle
+| [†](README.md#linux-only)FD 3              | A third open file descriptor
+| [‡](README.md#windows-only)`CNB_EXEC_D_HANDLE`            | Hex representation of an open file handle
```

Comment:
```suggestion
| [†](README.md#linux-only)FD 3              | A third open file descriptor
| [‡](README.md#windows-only) `<handle>`  | An additional open handle
| [‡](README.md#windows-only)`CNB_EXEC_D_HANDLE` | Hexidecimal number for `<handle>`
```
Couldn't grab the line above in the suggestion (to fix spaces) so spacing will need to be fixed if you accept suggestion.